src/explore-education-statistics-api-docs/source/templates/reference/endpoint_index.html.md.erb

```diff
@@ -0,0 +1,20 @@
+---
+last_reviewed_on: 2024-11-15
+review_in: 24 months
+---
+
+# <%= title %>
+
+This section lists all the endpoints available on the explore education statistics (EES) <%= version %> API.
```

Comment:
Capitalise the first "e" again?

Reply:
See comment: https://github.com/dfe-analytical-services/explore-education-statistics/pull/5397#discussion_r1853787613